common/src/main/java/com/microsoft/identity/common/internal/cache/MsalAccountCredentialCache.java

```diff
@@ -0,0 +1,46 @@
+package com.microsoft.identity.common.internal.cache;
+
+import android.content.Context;
+
+import com.microsoft.identity.common.Account;
+import com.microsoft.identity.common.internal.providers.oauth2.AuthorizationRequest;
+import com.microsoft.identity.common.internal.providers.oauth2.OAuth2Strategy;
+import com.microsoft.identity.common.internal.providers.oauth2.OAuth2TokenCache;
+import com.microsoft.identity.common.internal.providers.oauth2.RefreshToken;
+import com.microsoft.identity.common.internal.providers.oauth2.TokenResponse;
+
+public class MsalAccountCredentialCache
+        extends OAuth2TokenCache
```

Comment:
Just logging a note from our conversation this morning: we should consider naming alternatives or a new interface for this cache in light of the fact that it's been genericized beyond OAuth2....

/cc @shoatman